### PR TITLE
Disallow JUnit assertions

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

File: gateway-ha/pom.xml [hunk 1]
```diff
@@ -568,6 +568,21 @@
                     </dependency>
                 </dependencies>
             </plugin>
+
+            <plugin>
```
Comment (mosabua) [inline gateway-ha/pom.xml:572]: Cant we just inherit that from airbase instead or is that not defined there? Also we need to make sure this adds and does not override other rules.

Comment (mosabua) [reply gateway-ha/pom.xml:572]: Also .. maybe this should go in pluginMgt in the parent

**Review by mosabua — COMMENTED** [review]
In principal I think this is great. Just have to look at impl and refine a bit maybe.

File: gateway-ha/pom.xml (hunk 1 above)
Comment (ebyhr) [reply gateway-ha/pom.xml]: Airbase doesn't have this rule https://github.com/airlift/airbase/blob/f9cb1aafcbe1392d45414ddecab186468e7e573c/airbase/pom.xml#L688-L738

> Also we need to make sure this adds and does not override other rules.

I already confirmed it's append.

**Review by mosabua — APPROVED** [review]
Lets ship this, probably worth adding to trino some time

**Pull request merged**